pyhelm/chartbuilder.py

```diff
@@ -236,6 +233,18 @@ def get_helm_chart(self):
         self._helm_chart = helm_chart
         return helm_chart
 
+    @staticmethod
+    def read_file(self, path):
+        '''
+        Open the file provided in `path` and strip any non-UTF8 characters.
+        Return back the cleaned content
+        '''
+        with codecs.open(path, encoding='utf-8', errors='ignore') as fd:
+            content = fd.read()
+            fd.close()
```

Comment:
There's no need to call ```fd.close()``` as the with block takes care of that.

Reply:
Thanks.. Amended now